daemon/algod/api/server/v2/delta_test.go

```diff
@@ -138,6 +140,10 @@ func TestDelta(t *testing.T) {
 	require.Equal(t, expAppDelta.Addr.String(), actAppDelta.Address)
 	require.Equal(t, expAppDelta.Params.Deleted, actAppDelta.AppDeleted)
 	require.Equal(t, len(original.KvMods), len(*converted.KvMods))
+	// sort the result so we have deterministic order
+	sort.Slice(*converted.KvMods, func(i, j int) bool {
+		return bytes.Compare(*(*converted.KvMods)[i].Key, *(*converted.KvMods)[j].Key) < 0
+	})
```

Comment:
There is also a `require.ElementsMatch` feature which claims to ignore the order

Reply:
Yeah I wasn't sure if that would do a deep equals or not.